docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -39,6 +39,586 @@ For more information about versions, build numbers, and Office Online Server, se
+
+> [!NOTE]
+> Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
+>
+> To use preview APIs, you must reference the **beta** library on the CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js and you may also need to join the Office Insider program to get a sufficiently recent Office build.
+
+Included in this release are more than 400 new APIs for Excel. The first table provides a concise summary of the APIs, while the subsequent tables gives a detailed list. Please try the new features and share your feedback with us.
+
+| Feature Area | Description | Relevant Objects/Articles |
+| --- | --- | --- |
+| Slicer | Insert and configure slicers to tables and PivotTables. | [Slicer](/javascript/api/excel/slicer) |
+| Comments | Add, edit, and delete comments. | [Comment](/javascript/api/excel/comment), [CommentCollection](/javascript/api/excel/commentcollection) |
+| Shapes | Insert, position, and format images, geometric shapes and text boxes. | [ShapeCollection](/javascript/api/excel/shapecollection) [Shape](/javascript/api/excel/shape) [GeometricShape](/javascript/api/excel/geometricshape)  [Image](/javascript/api/excel/image) |
+| New Charts | Explore our new supported chart types: maps, box and whisker, waterfall, sunburst, pareto. and funnel. | [Chart](/javascript/api/excel/charttype) |
+| Auto Filter | Add filters to ranges. | [AutoFilter](/javascript/api/excel/autofilter) |
+| Areas | Support for discontinuous ranges. | [RangeAreas](/javascript/api/excel/rangeareas) |
+| Special Cells | Get cells containing dates, comments, or formulas within a range. | [Range](/javascript/api/excel/range#getspecialcells-celltype--cellvaluetype-)|
+| Find | Find values or formulas within a range or worksheet. | [Range](/javascript/api/excel/range#find-text--criteria-)[Worksheet](/javascript/api/excel/worksheet#findall-text--criteria-) |
+| Copy Paste | Copy values, formats, and formulas from one range to another. | [Range](/javascript/api/excel/range#copyfrom-sourcerange--copytype--skipblanks--transpose-) |
+| RangeFormat | New capabilities with range formats. | [Range](/javascript/api/excel/rangeformat) |
+| Workbook Save, Close | Save and close workbooks.  | [Workbook](/javascript/api/excel/workbook) |
+| Insert Workbook | Insert one workbook into another.  | [Workbook](/javascript/api/excel/worksheetcollection) |
+| Calculation | Greater control over the Excel calculation engine. | [Application](/javascript/api/excel/application) |
+
+|Class|Fields|Description|
```

Comment:
This table isn't being displayed correctly on the staging site: https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets?branch=AlexJ-WhatsNew#excel-javascript-preview-api-set

![image](https://user-images.githubusercontent.com/3019529/52879191-64652280-3113-11e9-9d5f-cc56ba061d02.png)

Some suggestions you might try to fix this:
* Add additional blank line(s) between this table and the preceding one.
* Add a sentence between this table and the preceding one, so that there aren't 2 tables back-to-back. ("The following table provides a detailed list of the APIs that are summarized in the preceding table.")
* Update the header row of the table to add spaces between pipes and words -> `| Class | Fields | Description |`